.gitignore

```diff
@@ -23,4 +23,7 @@
 hs_err_pid*
 
 # Ignore all file in the "target" directory
-target/
+target/*
+!target/classes
```

Comment:
A co tu się stało, czemu nagle taki rozgardiasz w `gitignore`? Czemu chcemy mieć klasy na repozytorium i czemu też pojawia się jakiś plik `.properties` w katalogu `target`?

Reply:
Plik w sumie nie potrzebny w repo zdalnym. Potrzebny jest mi do uruchamiania aplikacji z pod eclipse. Kilka dni temu zorientowałem się dlaczego on tutaj jest potrzebny, a gdzie faktycznie powinien być, żeby np. za pomocą skryptu poprawnie uruchomiła się aplikacja. A że nie chciało mi działać z eclipse, to myślałem że to główne miejsce tego pliku. Dopiero jak zacząłem uruchamiać aplikację po za eclipse, to zdałem sobie sprawę o co chodzi z tymi plikami.

Reply:
Ogólnie `target` to ścieżka, pod którą Eclipse umieszcza zbudowane źródła programu w plikach `.class` itp., nie potrzebujesz tego umieszczać na repo

Reply:
I w ogóle nie zrozumiałem Twojej odpowiedzi szczerze powiedziawszy

Reply:
To poprawię w developie, bo to chyba dotyczy rzeczy z developa, a nie readme.

> Ogólnie target to ścieżka, pod którą Eclipse umieszcza zbudowane źródła programu w plikach .class itp., nie potrzebujesz tego umieszczać na repo

Tak :) Teraz to wiem.

Chodziło mi o to, że jak miałem ten plik po za katalogiem target to mi się w eclipse nie uruchamiało. I myślałem że ten plik jest potrzebny dla aplikacji, że musi być w target. Dopiero jak zacząłem w konsoli odpalać aplikację poza eclipse, to zdałem sobie sprawę, że target nie jest potrzebny, bo on się tworzy podczas budowy i plik properties potrzebny jest w resources.

Tak czy inaczej, poprawię to, bo już wiem o co chodzi :)

Reply:
Teraz sprawdziłem i wygląda na to że chyba już to poprawiłem, bo w .gitignore jest, żeby ignorować cały katalog target.

Reply:
Ale cały czas widzę na repozytorium katalog target...

Reply:
Zrobione, poprawiłem wpis w .gitignore, po commicie folder target został usunięty.

Reply:
Cały czas folder target jest na repo